frb_codegen/src/lib.rs

```diff
@@ -66,11 +66,13 @@ pub fn frb_codegen_multi(
     let raw_ir_file = config.get_ir_file()?;
 
     info!("Phase: Transform IR");
-    let ir_file = transformer::transform(raw_ir_file);
+    let mut ir_file = transformer::transform(raw_ir_file);
 
     info!("Phase: Generate Rust code");
     let generated_rust = generate_rust_code(config, &ir_file)?;
 
+    ir_file.remove_raw_prefix();
```

Comment:
Ha you are pointing to an interesting place: `transformer::transform`. So, shall we do the remove-prefix thing inside the `transform`? Because a transform does nothing but transforming sth in the ir, thus our case perfectly fit it

Reply:
I’d agree with you here 100%, but in rust it should stay `r#`, so unless I add this as a transformer function, idk sth. like `transformer::convert_for_dart(ir_file)`, I cannot do it before

Reply:
should I add it for the one function?

Reply:
Ah I see. Then the IR seen by rust and dart differs...

Then is it possible we do it somewhere else. For example, in IrStruct, when we are reading a field name, we give two function like `rust_field_name() { self.naive_fieldname }` and `fn dart_field_name() {self.naivefieldname.strip_prefix()}`.  You know, just like how today we read rust type names vs dart type names vs other things.

Reply:
puuuuh okay, but then most functions right now are generalised and then they’ll also need two individual implementations, don’t they?

Am not 100% sure, haven’t given it a proper look yet

Reply:
No? Only the function that reads *field name of a struct* so I guess should change <20 lines. If too many, just tell me!

Reply:
Ok will check it out

Reply:
I might have worked up a bit of magic or I just realised how damn stupid I was before, you choose